src/oop/oop2.py

```diff
@@ -18,6 +19,12 @@ def __init__(self, num_wheels):
 # Override the drive() method in Motorcycle so that it returns "BRAAAP!!"
 
 # TODO
+class Motorcycle(GroundVehicle):
+    def __init__(self):
+        super().__init__(num_wheels=2)
```

Comment:
You can just pass "2" in without redefining "num_wheels" but I do agree that this can be more readable.